styles.css

```diff
@@ -15,5 +20,63 @@ border-color: hotpink;
 
 h1 {
     color: hotpink;
-    font-family: 'Arial'
-}
\ No newline at end of file
+}
+
+.question-card {
+    display: flex;
+    position: relative;
+    margin: 35px auto 0 auto;
+    flex-direction: column;
+    align-items: center;
+    width: 70%;
+    background-color: peachpuff;
+
+border: solid;
+border-color: hotpink;
+}
+.question {
+    padding-top: 40px;
+    color: hotpink;
+    font-size: 20px;
+    text-align: center;
+}
+
+.answer-button {
+    padding: 5px;
+    margin: 40px auto 0 auto;
+    background-color: peachpuff;
+    border-color: hotpink;
+    color: hotpink;
+    font-size: 20px;
+}
+
```

Comment:
How about changing the color of your answer-button on hover?
Smth like:
.answer-button:hover { color: your-color; }